.gitignore

```diff
@@ -1,5 +1,23 @@
+# Composer Dependencies
```

Comment:
Can we throw an (empty) heading at the top for `# Project-specific Ignores`? This is a skeleton after all, and it should encourage the format we've agreed on (relevant stuff at the top, generic stuff below.)